actions.yaml

```diff
@@ -6,8 +6,9 @@ build-on-commit:
             type: string
         reference-bundle:
             description: |
-                Optional charm store URL for a bundle to use to test the given
+                Charm store URL for a bundle to use to test the given
                 charm. For example, "~charmers/mediawiki-single".
+                If this is not provided, the charm must it in its tests.yaml.
```

Comment:
Shouldn't we have reference-bundle required in the build-on-release? There is also something missing in "the charm must it in its tests.yaml" (set it?).